### PR TITLE
Add flake8 and python black formatting Jupyterlab extensions

### PR DESCRIPTION
**Pull request by s-block**

### CODE REVIEW

File: datascience-notebook/Dockerfile [hunk 1]
```diff
@@ -10,11 +10,18 @@ ENV CHOWN_HOME=no
...
     && apt-get update \
     && apt-get install -y $(cat /tmp/apt_packages) \
     && rm -rf /var/lib/apt/lists/*
 
+RUN npm install jupyterlab-flake8
+
+RUN jupyter labextension install @ryantam626/jupyterlab_code_formatter
+RUN jupyter serverextension enable --py jupyterlab_code_formatter
+
+RUN jupyter labextension install jupyterlab-flake8
```
Comment (andyhd) [inline datascience-notebook/Dockerfile]: This is a lot of `RUN` layers. Can we combine them with `&&`?

File: datascience-notebook/Dockerfile [hunk 2]
```diff
@@ -10,11 +10,18 @@ ENV CHOWN_HOME=no
 
 COPY ./files/apt_packages /tmp/
```
Comment (andyhd) [inline datascience-notebook/Dockerfile]: I know this isn't in the scope of this PR, but do you need to copy the file if you're just `cat`ing the contents to `apt-get install`?

File: datascience-notebook/Dockerfile (hunk 1 above)
Comment (andyhd) [reply datascience-notebook/Dockerfile]: Or you could do a multi-stage Dockerfile and just copy the results of installing the formatter and flake8 into the final image

Comment (s-block) [reply datascience-notebook/Dockerfile]: Added the lines as one RUN command

**Pull request merged**